### DISCUSSION TITLE
supporting out of the box JSON Path with Mojolicious

### DISCUSSION THREAD

**Discussion by b03cmans** [mojolicious/mojo#1973] [Ideas]
How about if **Mojo::UserAgent** responses has out of the box and built-in capability to select and extract JSON data from JSON responses using **JSON Path** in addition to the JSON pointer and Mojo::DOM which it already have today , this really will make Mojo::UserAgent more powerful  when dealing when with JSON responses.

**Comment by kraih** [comment]
Is there a proper spec for JSON Path yet?

**Reply by b03cmans** [reply]
This is no official spec for JSON Path yet .
this is the up to date version of the draft https://datatracker.ietf.org/doc/draft-ietf-jsonpath-base/05/
and this is the initial paper https://goessner.net/articles/JsonPath/  which most of the implementation using
